src/interface/config/IServiceConfig.ts

```diff
@@ -3,4 +3,5 @@ import IRequestConfig from "./IRequestConfig";
 export default interface IServiceConfig {
     serviceConfig: IRequestConfig;
     defaultLogConfig?: any;
+    serializer?<T>(log: T): any;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type. ([typescript:S4204](https://sonarcloud.io/organizations/alexeypopovua-github/rules#rule_key=typescript%3AS4204))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=advanced-logger&pullRequest=75&issues=AWfltDiNrt4Kct_4x7ps&open=AWfltDiNrt4Kct_4x7ps)

[ISSUE_KEY]: AWfltDiNrt4Kct_4x7ps